data/simple/bergamot.in

```diff
@@ -1 +1,5 @@
-The Bergamot project will add and improve client-side machine translation in a web browser. Unlike current cloud-based options, running directly on users' machines empowers citizens to preserve their privacy and increases the uptake of language technologies in Europe in various sectors that require confidentiality. Free software integrated with an open-source web browser, such as Mozilla Firefox, will enable bottom-up adoption by non-experts, resulting in cost savings for private and public sector users who would otherwise procure translation or operate monolingually. Bergamot is a consortium coordinated by the University of Edinburgh with partners Charles University in Prague, the University of Sheffield, University of Tartu, and Mozilla.
+The Bergamot project will add and improve client-side machine translation in a web browser.
+
+Unlike current cloud-based options, running directly on users’ machines empowers citizens to preserve their privacy and increases the uptake of language technologies in Europe in various sectors that require confidentiality. Free software integrated with an open-source web browser, such as Mozilla Firefox, will enable bottom-up adoption by non-experts, resulting in cost savings for private and public sector users who would otherwise procure translation or operate monolingually.
```

Comment:
If you add 3-4 mores whitespaces in the beginning of the sentence "Free software integrated", you will have a complete test case 👍

Reply:
I get where you're going at, I converted the text to some old word processor paragraph form. 3-4 whitespace are required at the beginning actually, "The Bergamot Project". Added a bit of whitespace buffer towards the end as well.